backend/tests/test_file_monitoring_websocket.py

```
@@ -0,0 +1,265 @@
+to subscribed clients.
+"""
+
+import pytest
+import asyncio
+import time
+from unittest.mock import AsyncMock, patch
+from uuid import uuid4
+
+from app.websocket.connection_manager import ConnectionManager
+from app.models.contracts import ConversationData
+from app.websocket.connection_manager import ConnectionManager
```

Ruff (0.12.2)

Redefinition of unused `ConnectionManager` from line 14

Remove definition: `ConnectionManager`

(F811)